protocols/v2/binary-sv2/binary-sv2/src/lib.rs

```diff
@@ -1,5 +1,10 @@
 // TODO unify errors from serde_sv2 and no-serde-sv2
-//
+
+#![no_std]
```

Comment:
why aren't we following this pattern here?
```
#![cfg_attr(feature = "no_std", no_std)]
```

Reply:
Because this crate is no_std compliant no matter what. No need to use a feature for it. KISS